test/container/module_activation_store.test.ts

```diff
@@ -5,69 +5,83 @@ import { interfaces } from "../../src/inversify"
 describe("ModuleActivationStore", () => {
   it("should remove handlers added by the module", () => {
```

Comment:
being picky - now that the serviceIdentifier is used perhaps these tests should have handlers for different service identifiers

Reply:
Right, I'll add it